diesel/src/mysql/types/numeric.rs

```diff
@@ -20,7 +20,7 @@ pub mod bigdecimal {
     }
 
     impl FromSql<Numeric, Mysql> for BigDecimal {
-        fn from_sql(bytes: Option<&<Mysql as Backend>::RawValue>) -> deserialize::Result<Self> {
+        fn from_sql<'a>(bytes: Option<<<Mysql as Backend>::RawValue as FamilyLt<'a>>::Out>) -> deserialize::Result<Self> {
```

Comment:
Just use the actual type here:
```suggestion
        fn from_sql<'a>(bytes: Option<&'a [u8]>) -> deserialize::Result<Self> {
```